testing/trino-tests/src/test/java/io/trino/tests/AbstractQueryAssertionsTest.java

```diff
@@ -92,11 +92,7 @@ public void testWrongType()
     {
         QueryAssert queryAssert = assertThat(query("SELECT X'001234'"));
         assertThatThrownBy(() -> queryAssert.matches("VALUES '001234'"))
-                .hasMessageContaining("[Output types] \n" +
-                        "Expecting:\n" +
-                        " <[varbinary]>\n" +
-                        "to be equal to:\n" +
-                        " <[varchar(6)]>");
+                .hasMessageContaining("[Output types] expected:<[var[char(6)]]> but was:<[var[binary]]>");
```

Comment:
unrelated to the rest of `Allow to reuse few testing servers` commit?

Reply:
It is related somehow. Adding testcontainers dependency updated assertion dependency and we got different message.

Reply:
I thought we're pinning assetions (assertj?) version explicitly. Pulling in testcontainers shouldn't change anything here.